assets/src/ba_data/python/bastd/game/easteregghunt.py

```diff
@@ -147,7 +147,7 @@ def _on_egg_player_collide(self) -> None:
                 player = (spaz.getplayer()
                           if hasattr(spaz, 'getplayer') else None)
                 if player and egg:
-                    player.get_team().gamedata['score'] += 1
```

Comment:
Note: the fact that this error didn't get picked up by type-checking ('make mypy' or 'make check') tells me that the type-checker doesn't have types for these variables here. See https://github.com/efroemling/ballistica/wiki/Knowledge-Nuggets#beware-the-any-type
Don't worry about it for now, but in the future it could be good to first get the type checker detecting the issue and then fixing the issue (that way the type checker would warn us if other issues popped up in this code later).
Adding an `assert isinstance(player, ba.Player) or player is None` would inform the type checker what player is supposed to be. (which you could then verify with reveal_type())